### PR TITLE
Weather app

### PR DESCRIPTION
**Pull request by MahmoudBakr23**
A weather detector application with javascript, node.js and OpenWeatherMap API
First: I created HTML elements which is gonna wrap our received data from the API
Second: I called out these elements in Javascript
Third: I added an event listener that triggers the async function when submitted
Fourth: The async function takes the input (search) value and add it to the upcoming fetched API call
Finally: I turn the received data into JSON so that I can use it and manipulate it the way I want.

### CODE REVIEW

**Comment by bellom** [comment]
> A weather detector application with javascript, node.js and openWeatherMap API

**You can improve on this PR description by including each task carried out while building project**

Example; 
image
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/

File: README.md [hunk 1]
```diff
@@ -1 +1,39 @@
...
+- Pure JavaScript
+- DOM properties and attributes
+- HTML & SCSS
+- openWeatherMap API
+
+## Getting Started
+To get a local copy of the repository please run the following commands on your terminal:
+- ```$ cd <folder>```
+- ```$ git clone https://github.com/MahmoudBakr23/weather-app.git ```
+- ```$ cd weather-app ```
```
Comment (bellom) [inline README.md:22]: Let add more instruction to these steps for people that are new to programming.
example; 
- run `npm start`
- open index.html

**Comment by MahmoudBakr23** [comment]
Thank you for the heads up, but I don't see a review yet!

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      img.classList.remove('d-none');
+      any.innerHTML = `
+                ${myData.sys.country}
+            `;
+      desc.innerHTML = `
+                The weather in your region is ${myData.weather[0].description.toUpperCase()}!
+            `;
+    }
+  }
+  getWeather();
```
Comment (bellom) [inline src/index.js]: Let's separate DOM related tasks from logic tasks.
example;
`mainTitle.innerHTML = ` is included in the main logic of your app.
You can create another method to handle DOM tasks.

image

https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @MahmoudBakr23  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Changes Required! ♻️ 

**Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Review by bellom — APPROVED** [review]
## Hi @MahmoudBakr23  :clap: 

**Great work making all the necessary corrections from the previous review.**

# Status: Approved 👍 

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit the project completion form.
Good luck in the next project :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**